04/rhys/test_usertweets.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+import csv
+import unittest
+from unittest.mock import patch
+
+
+from tweets import TWEETS  # mock data
+from usertweets import UserTweets, NUM_TWEETS
+
+HANDLE = 'pybites'
+MAX_ID = '819831370113351680'
+
+Tweet = namedtuple('Tweet', ['id_str', 'created_at', 'text'])
+
+
+def read_csv(fname):
+    with open(fname) as f:
+        has_header = csv.Sniffer().has_header(f.readline())
+        f.seek(0)
+        r = csv.reader(f)
+        if has_header:
+            next(r, None)  # skip the header
+        return [Tweet(*tw) for tw in r]  # list(r)
+
+
+class TestUserTweets(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+        with patch('tweepy.API') as mock_timeline:
```

Comment:
Cause of issue + our oversight, fixed on master (template) branch: https://github.com/pybites/challenges/commit/7221ee273dc67978bfbbbb3959a858310e6e9bb6 - thanks.